### PR TITLE
Request new AFL++ experiment

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

File: service/experiment-requests.yaml [hunk 1]
```diff
@@ -19,6 +19,14 @@
 # are still testing this feature. You should request an experiment by contacting
 # us as you normally do.
 
+- experiment: 2020-06-17
```
Comment (inferno-chromium) [inline service/experiment-requests.yaml:22]: any aflplusplus suffix or just general date ?

**Comment by jonathanmetzman** [comment]
TBR

**Comment by vanhauser-thc** [comment]
uh this feature works now? :)

**Comment by jonathanmetzman** [comment]
> uh this feature works now? :)

It's still not for public consumption but yes.

File: service/experiment-requests.yaml (hunk 1 above)
Comment (jonathanmetzman) [reply service/experiment-requests.yaml:22]: Date seems good tome.

**Pull request merged**